### DISCUSSION TITLE
Scheduling events with seconds in time

### DISCUSSION THREAD

**Discussion by acidkewpie** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#156] [Ideas]
As much as I see it not being useful that often, allowing timings to the second could be very useful in a few situations. I have the schedule set up to turn the bubbles on for a minute at 2am, after the pumps have come back on after an evening of quiet. The tub is in our conservatory next to our lounge, so it's already awesome having the pump turn off automatically for the evening when we're watching TV nearby. But of course 8 hours of no pump means any dirt the water can settle, so a blast of bubbles churns up the water something lovely to let the pump filter as well as possible over night. A minute feels like quite a long time in this scenario, only need 5 to 10 seconds to get the job done with minimal disturbance, and can then do it twice, three times overnight in total for less than one minute. Given the repeat schedule is done on seconds, it does feel a little wonky that we can't specific minutes in the event times..?

**Comment by visualapproach** [comment]
Hi there! The device accepts seconds but unfortunately the browsers just show minutes. If you show the console in the browser you should be able to send commands by the second, albeit somewhat cumbersome. I'll see if I can do something next weekend.

**Comment by visualapproach** [comment]
@TheRocketSurgeon I solved this now. Download the updated file "config.html" from development branch. Easiest method to upload the file to the ESP8266 is to visit page "yourdeviceIP/upload.html". Select the new "config.html" and click upload. You can now set seconds in the time selector. (Doesn't seem to work in Safari browser on the iPhone)

**Reply by acidkewpie** [reply]
Cracking, updated. :)
